### DISCUSSION TITLE
文档中没有写明组件的导入名

### DISCUSSION THREAD

**Discussion by JmyFilm** [tusen-ai/naive-ui#2701] [Ideas]
查看组件文档时，发现组件中没有写明该组件需要调用的模块

当按需导入时，难以确认需导入的组件模块名字，和需导入的组件~

感觉可以名字后面写一下🤣
![image](https://user-images.githubusercontent.com/42228457/160666774-5e090492-9b2b-48c3-a94d-a4f04edda77c.png)

**Comment by 07akioni** [comment]
这个规律我感觉还是比较明确的，会写一个就能找到其他的，可以看看其他人有没有这个诉求
